.github/workflows/test.yml

```diff
@@ -246,7 +246,7 @@ jobs:
       - run: |
           make mockgen -j $(nproc)
           hack/tree_status.sh
-      - run: make testunit
+      - run: sudo PATH="$PATH" make testunit
```

Comment:
(commenting on the commit message, not the change itself)

Would be nice to explain why sudo is needed ("test case XXX added by an earlier commit performs mounts, which require sudo" or somesuch).

Reply:
Also, this probably belongs to the commit that added the test that needs sudo.

Reply:
> Also, this probably belongs to the commit that added the test that needs sudo.

That is, unless you have added a check that skips the test as non-root.

Reply:
I moved this commit before the other